backend/typescript/prisma/schema.prisma

```diff
@@ -12,11 +12,21 @@ model test {
 }
 
 model users {
-   id        String @id @default(auto()) @map("_id") @db.ObjectId
-   v         Int    @map("__v")
-   authId    String
-   email     String
-   firstName String
-   lastName  String
-   role      String
- }
\ No newline at end of file
+  id        String @id @default(auto()) @map("_id") @db.ObjectId
+  v         Int    @map("__v")
+  authId    String
+  email     String
+  firstName String
+  lastName  String
+  role      String
+}
+
+model volunteerSignUp {
+  id        String @id @default(auto()) @map("_id") @db.ObjectId
+  v         Int    @map("__v")
+  email     String
+  firstName String
+  lastName  String
+  admin_id  String
```

Comment:
admin_id should be a related field that pulls the id of whoever manages that specific volunteer so there should be a relation back to the id field in users table.